### PR TITLE
BL-12117 Simplify 'Save BloomPUB File' panel

### PR DESCRIPTION
**Pull request by gmartin7**
This change is [Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5781)

### CODE REVIEW

**Review by andrew-polk — CHANGES_REQUESTED** [review]
> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5781)* status: all files reviewed, 2 unresolved discussions (waiting on @gmartin7)

___
*[`DistFiles/localization/en/Bloom.xlf` line 3569 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5781#-NTpK9lp225KA_seZiLb:-NTpK9lp225KA_seZiLc:bcntm37) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/6d49c073df185f92a1f7c2b0dfaec0b31624f581/DistFiles/localization/en/Bloom.xlf#L3569)):*
> ```XML
>       </trans-unit>
>       <trans-unit id="PublishTab.Android.BloomPUB.Hint.Heading" sil:dynamic="true">
>         
> ```

These should be `<note>Obsolete as of Bloom 5.5</note>` like others here. Then translators can see that it is going away.
___
*[`src/BloomBrowserUI/publish/ReaderPublish/MethodChooser.tsx` line 43 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5781#-NTpKVaCE0RM4nTtWw2u:-NTpKVaCE0RM4nTtWw2v:bi9md9n) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/6d49c073df185f92a1f7c2b0dfaec0b31624f581/src/BloomBrowserUI/publish/ReaderPublish/MethodChooser.tsx#L43)):*
> ```XML
>             css={css`
>                 font-size: 1rem;
>                 margin-top: 2px; // to line up correctly with the radio button
> ```

What changed that made this necessary? (Or are you fixing an existing bug?)

The main reason I'm asking is if we need to do this tweak here, do we (or why don't we) need to do it in RadioGroup?

**Review by gmartin7 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5781)* status: 2 of 3 files reviewed, 2 unresolved discussions (waiting on @andrew-polk)

___
*[`src/BloomBrowserUI/publish/ReaderPublish/MethodChooser.tsx` line 43 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5781#-NTpKVaCE0RM4nTtWw2u:-NTsq3jN1zK3v4jh7NLX:b-uw4sy2) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/6d49c073df185f92a1f7c2b0dfaec0b31624f581/src/BloomBrowserUI/publish/ReaderPublish/MethodChooser.tsx#L43)):*
<details><summary><i>Previously, gmartin7 (Gordon Martin) wrote…</i></summary><blockquote>

It's related to using a larger font-size. The default before was something like 0.87rem? I don't remember exactly. But with the Typography element added and the larger size, it didn't line up vertically with the actual radio button quite. (We were just passing the localized string here before, but the text size needed to be larger.)
</blockquote></details>

In other words, I don't think it would be helpful to adjust spacing in the RadioGroup.

**Review by andrew-polk — APPROVED** [review]
> Reviewed 1 of 1 files at r2, 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5781)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @gmartin7)

**Review by gmartin7 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5781)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @gmartin7)

___
*[`src/BloomBrowserUI/publish/ReaderPublish/MethodChooser.tsx` line 43 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5781#-NTpKVaCE0RM4nTtWw2u:-NTtO-vC5iGkIBc8y-dQ:b2vquul) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/6d49c073df185f92a1f7c2b0dfaec0b31624f581/src/BloomBrowserUI/publish/ReaderPublish/MethodChooser.tsx#L43)):*
<details><summary><i>Previously, gmartin7 (Gordon Martin) wrote…</i></summary><blockquote>

In other words, I don't think it would be helpful to adjust spacing in the RadioGroup.
</blockquote></details>

In fact, I went and looked and RadioGroup is already doing all the normal flexbox things to align the button and label vertically. But there doesn't seem to be any way to do Emotion on the RadioGroup and affect the label font-size, so I kept the Typography solution and just removed the 2px adjustment. Looks not bad.

**Comment by gmartin7** [comment]
Andrew merged this, but then it got in a weird state where it thought the PR was still open. I deleted it from GitKraken.
